File: CS/Q147080/Tab/MyTabStylesRegistrator.cs
using DevExpress.XtraTab.Registrator;
using DxSample.Tab.Styles;

namespace DxSample.Tab {
    public static class MyTabStylesRegistrator {
        public static void Register() {
            PaintStyleCollection.DefaultPaintStyles.Remove("Flat");
            PaintStyleCollection.DefaultPaintStyles.Add(new MyFlatViewInfoRegistrator());
            PaintStyleCollection.DefaultPaintStyles.Remove("Office2003");
            PaintStyleCollection.DefaultPaintStyles.Add(new MyOffice2003ViewInfoRegistrator());
            PaintStyleCollection.DefaultPaintStyles.Remove("WindowsXP");
            PaintStyleCollection.DefaultPaintStyles.Add(new MyWindowsXPViewInfoRegistrator());
            PaintStyleCollection.DefaultPaintStyles.Remove("Skin");
            PaintStyleCollection.DefaultPaintStyles.Add(new MySkinViewInfoRegistrator());
        }
    }
}

File: VB/Q147080/Tab/MyTabStylesRegistrator.vb
Imports DevExpress.XtraTab.Registrator
Imports DxSample.Tab.Styles

Namespace DxSample.Tab
    Public NotInheritable Class MyTabStylesRegistrator

        Private Sub New()
        End Sub

        Public Shared Sub Register()
            PaintStyleCollection.DefaultPaintStyles.Remove("Flat")
            PaintStyleCollection.DefaultPaintStyles.Add(New MyFlatViewInfoRegistrator())
            PaintStyleCollection.DefaultPaintStyles.Remove("Office2003")
            PaintStyleCollection.DefaultPaintStyles.Add(New MyOffice2003ViewInfoRegistrator())
            PaintStyleCollection.DefaultPaintStyles.Remove("WindowsXP")
            PaintStyleCollection.DefaultPaintStyles.Add(New MyWindowsXPViewInfoRegistrator())
            PaintStyleCollection.DefaultPaintStyles.Remove("Skin")
            PaintStyleCollection.DefaultPaintStyles.Add(New MySkinViewInfoRegistrator())
        End Sub
    End Class
End Namespace
